File: 10/AP/16/Soal72.java
public class Soal72{

    public static void main(String[] args)
    {
        int[] list = {11, 35, 6, 0};
        System.out.println(mystery(list));
    }

    public static int mystery(int[] list)
    {
        int sum = 0;
        for (int i = 0; i < list.length; i++)
        {
            sum += list[i];
        }
        return sum/list.length;
    }
}

File: 10/INFORMATIKA/C++/10-2/11/soal72.cpp
#include <iostream>
#include <vector>
using namespace std;

int mystery(vector<int> &list)
{   
    int sum = 0;
    for (int i = 0; i < list.size(); i++)
    {
        sum += list[i];
    }
    return sum / list.size();
}

int main()
{
    vector<int> list;
    list.push_back(11);
    list.push_back(35);
    list.push_back(6);
    list.push_back(0);
    cout << mystery(list) << endl;
}